r1.sh

```diff
@@ -65,20 +75,38 @@ python3 -m venv venv
 source venv/bin/activate
 
 chmod +x mtkbootcmd.py
+# Make `mtkbootcmd.py` use "${ESCAPED_PWD}"/venv/bin/python.
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    # Sed is different on mac.
```

Comment:
Are you able to use mtkclient reliably on macOS?

Reply:
No, I just didn't want to break the script if you do support mac down the road ^^'